packages/account/src/predicate/predicate.ts

```diff
@@ -47,7 +47,7 @@ export class Predicate<
 > extends Account {
   bytes: Uint8Array;
   predicateData: TData = [] as unknown as TData;
-  interface: Interface;
+  interface: AbiCoder;
```

Comment:
```suggestion
  abiCoder: AbiCoder;
```

and all the other places where this renaming would break them.